core-bundle/src/Messenger/WebWorker.php

```diff
@@ -146,10 +146,27 @@ private function processTransport(string $transportName, float $stopTime): void
 
         $input = new ArrayInput($inputParameters);
 
+        // The ConsumeMessagesCommand uses SymfonyStyle() which itself tries to
+        // automatically detect the optimal width and height of the terminal to adjust
+        // its output. For this, it uses the "stty" command and only does this, if the
+        // proc_open() function exists. So if the PHP setup is configured to support
+        // proc_open() but the "stty" process call fails for permission issues, the
+        // WebWorker will report PHP errors. We can prevent this from happening by
+        // defining the dimensions for the terminal directly as it is irrelevant for us
+        // anyway - we're using NullOutput.
+        $columnsBefore = (string) getenv('COLUMNS');
+        $linesBefore = (string) getenv('LINES');
+        putenv('COLUMNS=80');
+        putenv('LINES=50');
```

Comment:
We only need to do this if `COLUMNS` or `LINES` is not already set, right?

```suggestion
        if ((int) $columnsBefore < 1 || (int) $linesBefore < 1) {
            putenv('COLUMNS=80');
            putenv('LINES=50');
        }
```